src/node_sqlite.cc

```diff
@@ -239,6 +256,175 @@ void DatabaseSync::Exec(const FunctionCallbackInfo<Value>& args) {
   CHECK_ERROR_OR_THROW(env->isolate(), db->connection_, r, SQLITE_OK, void());
 }
 
+void DatabaseSync::CreateSession(const FunctionCallbackInfo<Value>& args) {
+  std::string table;
+  std::string dbName = "main";
+
+  Environment* env = Environment::GetCurrent(args);
+  if (args.Length() > 0) {
+    if (!args[0]->IsObject()) {
+      node::THROW_ERR_INVALID_ARG_TYPE(
+          env->isolate(), "The \"options\" argument must be an object.");
+      return;
+    }
+
+    Local<Object> options = args[0].As<Object>();
+
+    Local<String> table_key =
+        String::NewFromUtf8(env->isolate(), "table", NewStringType::kNormal)
+            .ToLocalChecked();
+    if (options->HasOwnProperty(env->context(), table_key).FromJust()) {
+      Local<Value> table_value =
+          options->Get(env->context(), table_key).ToLocalChecked();
```

Comment:
Nit: it is generally preferable to avoid use of `ToLocalChecked()` if possible. Something like

```suggestion
      Local<Value> table_value;
      if (!options->Get(env->context(), table_key).ToLocal(&table_value)) {
       return;
      }
```